src/java/org/apache/cassandra/db/compaction/AbstractCompactionTask.java

```diff
@@ -205,13 +200,13 @@ public void addObserver(CompactionObserver compObserver)
     }
 
     @VisibleForTesting
-    public List<CompactionObserver> getCompObservers()
+    List<CompactionObserver> getCompObservers()
     {
         return compObservers;
     }
 
     @VisibleForTesting
-    public LifecycleTransaction transaction()
+    ILifecycleTransaction getTransaction()
```

Comment:
can we make this public ?
we use this in CNDB: https://github.com/riptano/cndb/blob/ebf8bbb77b8b1be7a13b9427b88e6c37be23dc47/cndb-tasks/src/main/java/com/datastax/cndb/compactor/CompactionTasksOrganizer.java#L673